package.json

```diff
@@ -65,21 +65,22 @@
     "redux-action-log": "^2.0.1",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
-    "redux-undo": "1.0.0-beta9-9-6",
+    "redux-undo": "1.0.0-beta9-9-7",
     "reselect": "^2.5.4",
+    "source-list-map": "2.0.0",
     "tether": "^2.0.0",
     "throttle-debounce": "^3.0.1",
-    "tslib": "^2.2.0",
+    "tslib": "^2.3.0",
     "vega": "4.0.0",
-    "vega-datasets": "^1.8.0",
+    "vega-datasets": "^2.2.0",
```

Comment:
Did you try voyager? I think a lot of fields have changed names here.

Reply:
I noticed that some of new vega dataset are csv files, but currently it is set to receive only json files (response => response.json()).  How should I add a csv reader?

Reply:
I'd stay with Vega datasets v1 for now.

Reply:
OK.  

In version 1.9 through 1.31 (latest of v1), these are introduced:
disasters.csv
co2-concentration.csv
weather.csv
us-unemployment.csv
windvectors.csv

earthquakes.json
us-state-capitals.json
uniform-2d.json
volcano.json
annual-precip.json
ohlc.json

Would you like me to:
1) Keep it at the original version, 1.8
2) Upgrade to the latest of v1.x, but make only json datasets accessible.

Reply:
Latest1.x is fine

Reply:
Upgraded to 1.31.0.

Only the following three were added because the others weren't in an array format, which is a data type requirement:

- US-state-capitals
- Uniform-2d
- OHLC


![image](https://user-images.githubusercontent.com/19829334/123351064-5b2dc100-d52a-11eb-8e34-b80ebca274af.png)